tests/rptest/tests/data_transforms_test.py

```diff
@@ -1164,34 +1165,62 @@ def test_manager_metrics_values(self):
         self.redpanda.set_cluster_config(
             {'data_transforms_logging_flush_interval_ms': 60 * 60 * 1000})
 
+        cfg = self._admin.get_cluster_config()
+
+        buf_capacity_B = cfg.get(
+            'data_transforms_logging_buffer_capacity_bytes', 0)
+        assert buf_capacity_B > 0, "Expected non-zero log buffer capacity"
+
+        max_log_line_B = cfg.get('data_transforms_logging_line_max_bytes', 0)
+        assert max_log_line_B > 0, "Expected non-zero max log line"
+
+        self.logger.debug(
+            f"Buffer capacity: {buf_capacity_B}B, Max log line: {max_log_line_B}B"
+        )
+
         it, ot = self.setup_identity_xform(
             self.topics[0],
             self.topics[1],
             name=f"logger-xform",
         )
 
         N_PER_TP = 32
+        k = self.random_ascii_string(128, 128)
+        min_log_usage_B = N_PER_TP * max_log_line_B
         for _ in range(0, N_PER_TP):
```

Comment:
Alternatively maybe it's simpler to just have a single partition in the topic, but this is fine.